src/smc-webapp/course/actions.ts

```diff
@@ -2612,9 +2612,7 @@ You can find the comments they made in the folders below.\
             return webapp_client.write_text_file_to_project({
               project_id: store.get("course_project_id"),
               path: target_path + `/GRADER - ${name.simple}.txt`,
-              content: `The student who did the peer grading is named ${
-                name.full
-              }.`,
+              content: `The student who did the peer grading is named ${name.full}.`,
```

Comment:
This has to be caused by a prettier version change, since there's no way that the previous formatting here was done manually...